src/Sylius/Bundle/LocaleBundle/Templating/Helper/LocaleHelper.php

```diff
@@ -37,9 +37,9 @@ public function __construct(LocaleConverterInterface $localeConverter)
     /**
      * {@inheritdoc}
      */
-    public function convertCodeToName($localeCode)
+    public function convertCodeToName($localeCode, $locale = 'en')
     {
-        return $this->localeConverter->convertCodeToName($localeCode);
+        return $this->localeConverter->convertCodeToName($localeCode, $locale);
```

Comment:
What do you think about making `$locale` null by default, injecting `%sylius_locale.locale%` (which should be the default locale of the application) here and using it is a fallback when locale is not passed?